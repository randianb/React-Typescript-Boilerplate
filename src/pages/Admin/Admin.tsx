import React from 'react';

class Admin extends React.Component {
  public render() {
    return (
      <main className="admin">
        <h1>Admin</h1>
      </main>
    );
  }
}

export default Admin;
