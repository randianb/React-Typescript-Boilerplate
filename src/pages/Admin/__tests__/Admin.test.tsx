import React from 'react';
import Admin from '../Admin';
import { render, cleanup } from 'react-testing-library';

afterEach(cleanup);

describe('Admin', () => {
  it('Renders Admin page', () => {
    const { container } = render(<Admin />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
