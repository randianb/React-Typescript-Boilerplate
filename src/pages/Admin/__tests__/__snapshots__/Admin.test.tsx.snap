// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin Renders Admin page 1`] = `
<main
  class="admin"
>
  <h1>
    Admin
  </h1>
</main>
`;
